### PR TITLE
Move %special variables to scope

### PR DESCRIPTION
**Pull request by imaustink**
Related to #304

### CODE REVIEW

**Comment by imaustink** [comment]
@phillipskevin Looks like master is failing and I don't understand why: https://travis-ci.org/canjs/can-stache-bindings/builds/293349636?utm_source=github_status&utm_medium=notification

File: can-stache-bindings.js [hunk 1]
```diff
@@ -468,25 +468,66 @@ var behaviors = {
...
-					"%viewModel": viewModel,
 					"%scope": data.scope,
-					"%context": data.scope._context,
-					"%arguments": arguments
-				},{
-					notContext: true
+					"%context": data.scope._context
+				};
+
+				Object.defineProperties(specialValues, {
```
Comment (phillipskevin) [inline can-stache-bindings.js]: Can you `steal-remove` this entire Object.defineProperties instead of just removing the dev warning?

**Review by phillipskevin — DISMISSED** [review]

**Comment by phillipskevin** [comment]
I'll look into the test failure on master. It looks like it was caused by `can-view-scope`: https://github.com/canjs/can-stache-bindings/issues/330.

**Review by phillipskevin — APPROVED** [review]
Let's wait to merge/release until Monday.

**Review by phillipskevin — APPROVED** [review]

**Pull request merged**